runtests.py

```diff
@@ -129,7 +129,7 @@ def main() -> None:
         exit(1)
 
     if not args:
-        args = DEFAULT_COMMANDS[:]
+        args = DEFAULT_COMMANDS.copy()
```

Comment:
This might be slower when compiled with mypyc.

Reply:
Here are some benchmarks I made comparing `x[:]`, `x.copy()`, and `list(x)` (I tried using the `perf_compare.py` file, but it just froze my computer during the compilation stage).

For each benchmark I ran 10 million iterations, repeated the test 3 times, and took the average. I did this for both the interpreted and compiled versions (currently using CPython 3.10.9, and Mypyc checked out from this PR). This is the benchmark code I used (which can probably be improved):

```python
import time

diff = 0.0
nums = [1, 2, 3, 4]

for _ in range(10_000_000):
    start = time.time()

    # <snippet to test here>

    end = time.time()
    diff += end - start

print(diff)
```

These are the stats for my machine (units are in seconds):

| Mode        | `x = nums[:]` | `x = nums.copy()` | `x = list(nums)` |
|-------------|--------|------------|-----------|
| Interpreted | 2.610  | 2.451      | 3.161     |
| Compiled    | 2.767  | 2.428      | 3.245     |

Interestingly, both `nums[:]` and `list(nums)` are slightly slower when compiled, and `nums.copy()` is fastest overall, including when compiled.